app/jobs/sms_sender_otp_job.rb

```diff
@@ -15,10 +15,6 @@ def send_otp(twilio_service, code, mobile)
   end
 
   def otp_message(code)
-    <<-END.strip_heredoc
-      You requested a secure one-time password to log in to your #{APP_NAME} Account.
-
-      Please enter this secure one-time password: #{code}
-    END
+    "#{code} is your #{APP_NAME} one-time password."
```

Comment:
Not sure we even need this method now, maybe just inline into the call-site?  (thats what i'm going in another PR I'm working on).